### DISCUSSION TITLE
What subtitle fornat does the cmd ver create, and how to create .srt from it?

### DISCUSSION THREAD

**Discussion by orangeduu** [openai/whisper#1457] [Q&A]
I am running whisper in command prompt, and am using the program to create subtitles that I'll eventually edit in another program. For that, I need to at least have a .srt file or similar that the program can recognize, so the time stamps can be used. The format of transcription command outputs is as so: 
> [00:00.000 --> 00:06.000]  How is the environment today within the city of New York?

Is there a way within command, or another application, that I can convert that into a recognizable subtitle format? I'm personally not familiar with what this style is called.

**Comment by glangford** [comment]
Note that there are different "outputs" - there is output to the console if you have `--verbose True`. This is informational only. The output in srt format is written to a file if you specify `--output_format srt`.

```
--output_format {txt,vtt,srt,tsv,json,all}, -f {txt,vtt,srt,tsv,json,all}
                        format of the output file; if not specified, all available formats will be produced (default: all)
```
